optimizely/event/processor.go

```diff
@@ -63,35 +64,69 @@ func (p *QueueingEventProcessor) StartTicker() {
 	}()
 }
 
+func (p *QueueingEventProcessor) Append(current *Batch, new Batch) bool {
+	if current.ProjectID == new.ProjectID &&
+		current.Revision == new.Revision {
+		visitors := append(current.Visitors, new.Visitors[0])
```

Comment:
How come we need to create a new batch object just to hold 1 visitor? Can we just accept a visitor as argument and append it?